dev/TEAL.md

```diff
@@ -338,6 +367,24 @@ Asset fields include `AssetHolding` and `AssetParam` fields that are used in `as
 | 8 | AssetReserve | []byte | Reserve address |
 | 9 | AssetFreeze | []byte | Freeze address |
 | 10 | AssetClawback | []byte | Clawback address |
+| 11 | AssetCreator | []byte | Creator address |
+
+
+**App Fields**
+
+App fields used in the `app_params_get` opcode.
+
+| Index | Name | Type | Notes |
+| --- | --- | --- | --- |
+| 0 | AppApprovalProgram | []byte | Bytecode of Approval Program |
```

Comment:
This allows to have very huge strings in the stack of a TEAL program.
Can this be an issue?

The `concat` opcode is limiting outputs to 4096 bytes which should avoid most issues.

But still, you could get 8096 byte string that can be hashed.
This can cost a bit more than usual...

Reply:
Actually, all opcodes are limited to 4096 now. So accessing a big program will fail immediately.  Not ideal, but it prevents your hash cost concern.

Reply:
Sounds good. Can you make it clear in the specs? (Maybe I missed it.)